Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM golang:1.18.1-alpine3.15 AS gobuild-base
-RUN apk add --no-cache \
-	git \
-	make
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.18-fips-cbl-mariner2.0 AS gobuild-base
+RUN tdnf clean all && tdnf install -y \
```

Comment:
I'm thinking we can use the following command to install packages on the container image

RUN tdnf check-update \
    && tdnf install -y \
        git \
        make \
    && tdnf clean all

Reply:
Reflected on both builder and runtime.